src/styles/main.scss

```diff
@@ -2,28 +2,63 @@
 @import "sanitize.css";
 @import "fontface.css";
 
+// Resets
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
 :root {
   --base-margin: 1.6rem;
 }
 
 body {
   color: $basicColor;
   // prettier-ignore
-  font: 14px "OpenSans-Regular", sans-serif;
+  font: 16px "OpenSans-Regular", sans-serif;
   min-height: 100vh;
-  background: top / contain url(images/background.svg) no-repeat;
+  line-height: 1;
+  -webkit-font-smoothing: antialiased;
 }
 
-h1 {
-  font-size: 2.2em;
-  text-align: center;
+ol,
+ul {
+  list-style: none;
 }
 
-h2 {
-  font-size: 2em;
+a {
+  color: inherit;
+  text-decoration: inherit;
 }
 
 ::placeholder {
   /* test for autoprefixer in postcss */
   color: antiquewhite;
 }
+
+//Global
+.center-text {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: $whitTextColor;
+}
+
+.background-image {
```

Comment:
Hardly ever this is class is global & reusable because there is specific height and image that fits only for 1 element on the whole project. Please move this class to local file of related component/element